content/en/guide/v10/typescript.md

```diff
@@ -258,72 +331,7 @@ const AppContext = createContext({
 // }>
 ```
 
-It also requires you to pass in all the properties you defined in the initial value:
-
-```tsx
-function App() {
-  // This one errors 💥 as we haven't defined theme
-  return (
-    <AppContext.Provider
-      value={{
-//    ~~~~~ 
-// 💥 Error: theme not defined
-        lang: "de",
-        authenticated: true
-      }}
-    >
-    {}
-      <ComponentThatUsesAppContext />
-    </AppContext.Provider>
-  );
-}
-```
```

Comment:
This is explanation about TypeScript `TypeError`.
I think it is unnecessary in Preact document.
Therefore, I removed this sentence.